packages/@blockprotocol/graph/src/types/block-graph.ts

```diff
@@ -17,15 +17,17 @@ import {
 } from "./ontology/entity-type.js";
 import { Subgraph, SubgraphRootTypes } from "./subgraph.js";
 
-export type BlockGraphProperties = {
+export type BlockGraphProperties<
+  RootType extends SubgraphRootTypes["entity"] = SubgraphRootTypes["entity"],
```

Comment:
Changed to simply `Entity` in this [commit](https://github.com/blockprotocol/blockprotocol/commit/4ad1ec609b204ee0a8cbaf070f873c1abeb9ef97)

In theory this a bit less safe because it makes it possible for `SubgraphRootTypes["entity']["element"]` to change type without this changing type. I can change it to `SubgraphRootTypes["entity']["element"]` if you think this is a potential issue

Reply:
The classic safety vs ergonomics question 😅 

I think the ergonomics and intuition is worth the tradeoff here, we don't want block developers being hit in the face with all the complexity straight off the bat. Thanks!